src/Modules/ModParser.lua

```diff
@@ -981,6 +982,7 @@ local modTagList = {
 	["per (%d+) evasion rating, up to (%d+)%%"] = function(num, _, limit) return { tag = { type = "PerStat", stat = "Evasion", div = num, limit = tonumber(limit), limitTotal = true } } end,
 	["per (%d+) maximum energy shield"] = function(num) return { tag = { type = "PerStat", stat = "EnergyShield", div = num } } end,
 	["per (%d+) maximum life"] = function(num) return { tag = { type = "PerStat", stat = "Life", div = num } } end,
+	["per (%d+) player maximum life"] = function(num) return { tag = { type = "PerStat", actor = "Player", stat = "Life", div = num, actor = "parent" } } end,
```

Comment:
`actor` is doubled and there is no `Player` actor

Reply:
Thanks, forgot to remove that bit after I was done